manageiq-operator/pkg/helpers/miq-components/util.go

```diff
@@ -52,3 +53,42 @@ func addResourceReqs(memLimit, memReq, cpuLimit, cpuReq string, c *corev1.Contai
 
 	return nil
 }
+
+func serviceMutateFn(service *corev1.Service, labels map[string]string, ports []corev1.ServicePort, selector map[string]string) controllerutil.MutateFn {
+
+	mutateFn := func() error {
+		service.ObjectMeta.Labels = labels
+		service.Spec.Ports = ports
+		service.Spec.Selector = selector
+		return nil
+	}
+
+	return mutateFn
+}
+
+func configMapMutateFn(configMap *corev1.ConfigMap, labels map[string]string, data map[string]string) controllerutil.MutateFn {
+
+	mutateFn := func() error {
+		configMap.ObjectMeta.Labels = labels
+		if len(data) > 0 {
+			configMap.Data = data
+		}
+		return nil
+	}
+
+	return mutateFn
+
```

Comment:
Bonus space not needed.

Even if gofmt says it's okay I would prefer this type of style be consistent. At least it should be the same in each source file.